ideascube/templates/ideascube/includes/cards/deus-ex-silicium.html

```diff
@@ -1,3 +1,2 @@
 {% load i18n %}
-{% trans "learn" as theme %}
-{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme_class="learn" name="Deus Ex Silicium" description="Vidéos dédiées à la science des semi-conducteurs" theme=theme %}
+{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme="learn" %}
```

Comment:
> I will not go through all those files and rewrite a patch only for a better commit splitting.

That's a shame, the story we tell with our RCS history is extremely important. :disappointed: